packages/grid/_modules_/grid/hooks/utils/useResizeContainer.ts

```diff
@@ -8,6 +9,15 @@ export function useResizeContainer(apiRef): (size: ElementSize) => void {
   const gridLogger = useLogger('useResizeContainer');
   const { autoHeight } = useGridSelector(apiRef, optionsSelector);
 
+  const debounceResize = React.useMemo(
+    () =>
+      debounce((size: ElementSize) => {
+        gridLogger.info('resized...', size);
+        apiRef.current.resize();
+      }, 60),
```

Comment:
Is there a reason why it is 60? I mean it looks like an arbitrary number.

Reply:
yes it's a bit arbitrary. We had a chat about it https://github.com/mui-org/material-ui-x/issues/799#issuecomment-753311355

To me >100 seems a bit high so I thought 60 was good. Happy to change?